projects/demo/src/components/control/index.ts

```diff
@@ -1,20 +1,42 @@
 import {NgIf} from '@angular/common';
-import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Directive,
+    inject,
+    Input,
+} from '@angular/core';
+import {NgControl} from '@angular/forms';
 import {TuiDocAPIItem} from '@taiga-ui/addon-doc';
 import {TuiTitle} from '@taiga-ui/core';
 
+@Directive({
+    standalone: true,
+    selector: 'input[tuiDisabled][formControl]',
```

Comment:
I already have this directive here:
https://github.com/taiga-family/taiga-ui/blob/main/projects/demo/src/utils/disabled.directive.ts

Reply:
Oops 😄

Reply:
Fixed https://github.com/taiga-family/taiga-ui/pull/10031/commits/21703a58cae39021aef32a9ecf4cb297188a55ec